BackgroundOperationComparison.md

```diff
@@ -34,4 +34,10 @@ Edge has publicly stated that service workers will be supported in the browser i
  - [Opera Background Process](https://dev.opera.com/extensions/architecture-overview/)
  - [Safari Extension Background operation](https://developer.apple.com/library/safari/documentation/Tools/Conceptual/SafariExtensionGuide/ExtensionsOverview/ExtensionsOverview.html#//apple_ref/doc/uid/TP40009977-CH15-SW1)
  - [Android Background Sync](https://developers.google.com/web/updates/2015/12/background-sync?hl=en)
- - [Mozilla Service Workers](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API/Using_Service_Workers)
\ No newline at end of file
+ - [Mozilla Service Workers](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API/Using_Service_Workers)
+
+**Conclusion**
+
+Both native apps and browser extensionsi are allowed to execute on the user's behalf because the user cared enough about the functionality to install it. In the terms of the Budget API, we're defining that as a level of engagement with the app/extension. Service workers can be installed without a similar up front engagement check to the user, but they do require a permission grant to perform any background processing.
```

Comment:
> require a permission grant

I'd emphasize that this is currently the case. The Budget API does not prohibit specs from solely relying on user engagement for certain operations. If a UA can make this decision on behalf of the user, for instance because they added a site to their homescreen, express permission may not be necessary for _safer_ operations.